tekton/resources/prow-config-cd.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: trigger-to-deploy-prow-config
+spec:
+  params:
+  - name: gitRepository
+    value: $(body.gitRepository)
+  - name: gitRevision
+    value: $(body.gitRevision)
+  - name: configPath
+    value: $(body.configPath)
+  - name: namespace
+    value: $(body.namespace)
+  - name: clusterResource
+    value: $(body.clusterResource)
+---
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: prow-config-deployer
+spec:
+  serviceAccountName: release-right-meow
+  triggers:
+    - name: trigger
+      binding:
+        name: trigger-to-deploy-prow-config
+      template:
+        name: deploy-prow-config
+---
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: deploy-prow-config
+spec:
+  params:
+  - name: gitRepository
+    description: URL of the repository that holds the prow configuration
+  - name: gitRevision
+    description: Git revision
+  - name: namespace
+    description: Namespace to deploy to in the target cluster
+  - name: configPath
+    description: Path in the git repo that holds prow configs
+  - name: clusterResource
+    description: Name of the cluster resource that points to the target cluster
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: git-source-$(uid)
+    spec:
+      type: git
+      params:
+      - name: revision
+        value: $(params.gitRevision)
+      - name: url
+        value: https://$(params.gitRepository)
+  - apiVersion: tekton.dev/v1alpha1
+    kind: TaskRun
+    metadata:
+      generateName: deploy-prow-config-$(uid)-
```

Comment:
hm if `uid` is unique, does `generateName` make sense? (im not sure myself!)

Reply:
Good point, I guess I was too lazy to go and understand how `uid` is produced by triggers. We could probably drop the generated name, but I'll check in triggers first.

Reply:
Done